components/Card.js

```diff
@@ -21,13 +21,29 @@ export default function Card({
         </DeleteButton>
       )}
       <p>Date: {date}</p>
-      {weight && <p>Weight: {weight}</p>}
-      {height && <p>Height: {height}</p>}
-      {feastTime && (
-        <p>
-          Nurse time: {feastTime.substr(0, 2)}:{feastTime.substr(2, 2)}:
-          {feastTime.substr(4, 2)}
-        </p>
+      {weight?.[0] && (
+        <ul>
+          {weight.map((weight, index) => (
+            <li key={index}>Weight: {weight} </li>
+          ))}
+        </ul>
+      )}
+      {height?.[0] && (
+        <ul>
+          {height.map((height, index) => (
+            <li key={index}>Height: {height} </li>
+          ))}
+        </ul>
+      )}
```

Comment:
did you know that if your array is empty you can still "map" over it? I won't produce an outcome of course, but not an error either. So you don't even have to check if the array has some values, just map over all items, if the array happens to be empty no list items will be displayed.

Reply:
I definitely  didn't know that. But then the ```<ul>``` will still show up. Is it ok to put everything into one big ```<ul>``` ?